components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.html

```diff
@@ -21,7 +21,8 @@
           <input
             chefInput
             type="text"
-            (keyup)="handleFilterKeyUp($event.target.value)"
+            #filter
+            (keyup)="handleFilterKeyUp(filter.value)"
```

Comment:
I updated this as a result of the documentation found here: https://angular.io/guide/user-input#!#type-the--*event*

which suggests that passing $event is dubious and can be simplified so that we are only getting what we need, using a reference to the element.

Reply:
Nice--thank you!